Automattic/wp-calypso
# Pricing page: Display 'Included Products' section in the bundle lightboxes.

#### Proposed Changes

* Include a new 'Included Products' section in the bundle lightboxes.
* Removed the 'Includes' and 'Benefits' sections in the bundle lightboxes.

**Complete Bundle**
<img width="1095" alt="Screen Shot 2022-12-01 at 6 17 30 PM" src="https://user-images.githubusercontent.com/56598660/205027355-c8c77334-894c-46d7-9e72-dddfa44092ca.png">


**Security Bundle 10GB**
<img width="1096" alt="Screen Shot 2022-12-01 at 6 17 11 PM" src="https://user-images.githubusercontent.com/56598660/205027453-ca7f1d70-084a-4faf-b69b-6ffdd20809db.png">

**Security Bundle 1TB**
<img width="1100" alt="Screen Shot 2022-12-01 at 6 17 19 PM" src="https://user-images.githubusercontent.com/56598660/205027415-f16a7b59-dd19-4a4d-ad2d-a9d33be3659c.png">



#### Testing Instructions

1. Run this branch by following the steps below (or you can use the Jetpack Cloud live link commented on this PR)
    * Run `git fetch && git checkout add/product-included-info-in-bundle-lightbox`
    * Run `yarn start-jetpack-cloud`

2. Goto http://jetpack.cloud.localhost:3000/pricing or use the Jetpack Cloud live link and append `/pricing`
3. Press any '**More about Complete**' link to display a product lightbox.
4. Confirm that the product included and descriptions are correct (refer to the image before).
5. Close the lightbox then press any '**More about Security**' link to display a product lightbox.
6. Confirm that the product included and descriptions are correct (refer to the image before).
7. Toggle to the 1TB variation.
8. Confirm that the product included and descriptions are correct (refer to the image before).

#### Pre-merge Checklist





Related to 1202858161751496-as-1203463248826782
